lib/ansible/plugins/lookup/oc.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+# (c) 2017, Kenneth D. Evensen <kevensen@redhat.com>
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: ocp
```

Comment:
documented name does not match lookup, in any case oc/ocp seem too short, something longer and more descriptive might help users find this plugin

Reply:
@bcoca Does **ocp_query** make sense?  **openshift_query**?  I'm open to suggestions.

Reply:
naming is hard ... i think `openshift` should be good enough and descriptive enough as a lookup